tests/playwright/Report.md

```diff
@@ -0,0 +1,176 @@
+
+- Console debug option which is very useful in an event when tests are failing anonymously.
+- Scenarios that span multiple pages, domains, and iframes
+- Auto-wait for elements to be ready before executing actions (like click, fill)
+- Intercept network activity for stubbing and mocking network requests
+- Seamless integration with Jest.
+
+**Sample Page screenshot code**
+
+This code snippet navigates to wikipedia.org in WebKit and saves the screenshot.
+
+```
+    const playwright = require('playwright');
+
+    (async () => {
+        const browser = await webkit.launch();
+        const context = await browser.newContext();
+        const page = await context.newPage();
+        await page.goto('https://www.wikipedia.org/');
+        await page.screenshot({ path: 'wikipedia-home-page.png' });
+        await browser.close();
+    })();
+```
+
+**Playwright and Jest**
+
+ [Jest](https://jestjs.io/) is a delightful JavaScript Testing Framework with a focus on simplicity. Playwright can be integrated into a project in two ways and one of them is by using the [jest-playwright](https://github.com/playwright-community/jest-playwright) which has rich features like:
+
+ - Multi-browser and device (like iPhones with given screen sizes) support
+ - jest-dev-server integration which can start your webserver like create-react-app before running the tests
+ - expect-playwright integration which provides common expect helper functions
+
+ jest-playwright is added to the jest configuration as a preset which makes all the features availbale to be used at your disposal. it was inspired by [jest-puppeteer](https://github.com/smooth-code/jest-puppeteer/blob/master/README.md#start-a-server).
+
+ **Example of jest-playwright.config.js**
+
+```
+    module.exports = {
+        browsers: [process.env.BROWSER || "chromium"],
+        devices: ["iPhone 6", "Pixel 2"]
+    }
+```
+
+Below are some great and useful functions that come with jest-playwright;
+
+Reset helper functions. This functions come in handy when you want to reset the browser or page for various resasons such as deleting cookies. It exposes functions like, `resetPage(), resetContext() and resetBrowser()`.
+Example usage of reset current browser
+
+```
+    beforeEach(async () => {
+        await jestPlaywright.resetBrowser()
+    })
+```
+
+Debug helper functions. This help in debugging test incase there are test failure. It uses the `jestPlaywrightDebug()`
+
+Example usage of Debug
+
+```
+    test.jestPlaywrightDebug('failed', async ({ page }) => {
+        await page.goto('https://www.wikipedia.org/')
+        const title = await page.title()
+        await expect(title).toBe('Nothing')
+    })
+```
+
+For more of these gems that come with jest-playwright, please visit the [Github Repo](https://github.com/playwright-community/jest-playwright)
+
+#### Playwright and WebdriverIO metrics
+
+**Statics on  WebdriverIO and Playwright Popularity**
+
+![Screenshot](statistics.png)
```

Comment:
If this screenshot's job is to compare webdriverio and playwright, then remove all other tools. They are just adding noise. It's actually hard to find webdriverio and playwright on the chart and compare them.